src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -298,6 +297,13 @@ void prepare_primitive_fusing::fuse_bias(program &p) {
                 const_dep.get_output_layout().count() != static_cast<size_t>(out_features)) {
                 continue;
             }
+            auto parent_out_pshape = node->get_dependency(0).get_output_layout().get_partial_shape();
```

Comment:
It would be better to use `non_const_dep_idx`
```suggestion
            auto parent_out_pshape = node->get_dependency(non_const_dep_idx ).get_output_layout().get_partial_shape();
```